src/core/server/ui_settings/ui_settings_config.ts

```diff
@@ -56,7 +57,7 @@ const configSchema = schema.object({
   overrides: schema.object({}, { unknowns: 'allow' }),
   defaults: schema.object({
     'theme:darkMode': schema.maybe(schema.boolean({ defaultValue: false })),
-    'theme:version': schema.maybe(schema.string({ defaultValue: 'v8' })),
+    'theme:version': schema.maybe(schema.string({ defaultValue: defaultThemeVersion })),
```

Comment:
Note: I'm not 100% how this is used, so continued to use the actual version strings instead of labels (like settings/theme.ts). I think this shouldn't make this worse, but if this is supposed to use the same values as the other file, lmk and will update.

Reply:
I donno how I feel about a dependency defining the default value.

Reply:
If the goal is the schema is the source of truth, it should be factored that way. Currently, the default is spread in multiple places and this really isn't factored to be the source of truth (a dependency needs this value). I can change this to be independent, but not sure if we're gaining more than we're losing. I'm not sure how this practically matters given how osd is built today, but maybe i'm missing something about how consumers use this setup.

Reply:
> If the goal is the schema is the source of truth, it should be factored that way

No piece of code should need to know what the default is; they should all rely on UISettings to get the theme that is active. If any other piece of code is making decisions about defaults, it is just bad logic and would need to change.

I will pull in @ashwin-pc who might know otherwise.

Reply:
@AMoo-Miki Correct me if im wrong but is you concern here that we are updating the config default from an import? or that we have multiple places where the default theme is referenced?

Setting the default value for the UI setting like this should be okay. It is after all referencing a static value, As for seting the default theme as a fallback in the other places seems unnecessary since the config should already have atleast the default value. 

Maybe that was out of scope for the PR, but why cant we keep just one source of truth for what the current theme is and let it also handle how to fallback to the default if a value is not set?

Reply:
> is you concern here that we are updating the config default from an import? or that we have multiple places where the default theme is referenced?

Both. I like that this PR tries to centralize it but we shouldn't even have to because everyone should fetch any such value from UISettings and since a default value is provided there, they should trust that it exists. Less importantly, it feel wrong to me that OSD itself doesn't own the default value and that it breaks the existing pattern.

Reply:
> but we shouldn't even have to because everyone should fetch any such value from UISettings and since a default value is provided there

A problem with this statement is that @osd/ui-shared-deps, and I think optimizer, actually need this value. So a dependency of OSD would require OSD to define the default. If we want the schema to be the source of truth, that's fine, but IMO it likely needs to be factored into a package so it can used that way (at that point OSD would be depending on a package for its defaults - IMO not a problem if that's its responsibility - also would make it easier to swap out defaults).

I actually don't even understand how this value is used - uisettingsclient takes the default value from value of the uisettings definition, not the schema, e.g. https://github.com/opensearch-project/OpenSearch-Dashboards/blob/a360b38506a2665e76059534328c8a86c2449cb8/src/core/public/ui_settings/ui_settings_client.ts#L100 (although a couple lines above it, a comment says to use the defaultValue property). 

Anyways, I'm fine reverting the change in this schema file as it makes not real difference today (I'm not changing the default), but I think